tests/installation/first_boot.pm

```diff
@@ -29,6 +29,13 @@ sub run() {
             return;
         }
         mouse_hide();
+        if (match_has_tag("dm-nousers")) {
+            #no users created during installation, logging in as root
+            send_key 'tab';    #set focus to "Not listed?"
+            send_key 'ret';
+            wait_idle;
```

Comment:
then please try to get rid and test if it works - at best consistently with a busy worker